packages/atomic/cypress/integration/facets-v1/numeric-facet/numeric-facet.cypress.ts

```diff
@@ -0,0 +1,651 @@
+              NumericFacetSelectors,
+              1
+            );
+            CommonFacetAssertions.assertNumberOfIdleLinkValues(
+              NumericFacetSelectors,
+              defaultNumberOfValues - 1
+            );
+          });
+
+          describe('versify analytics', () => {
+            before(setupSecondLinkValue);
+            NumericFacetAssertions.assertLogNumericFacetSelect(
+              field,
+              secondSelectionIndex
+            );
+          });
+        });
+      });
+    });
+  });
+
+  describe('with custom ranges generate', () => {
+    const numberOfCustomRanges = numericRanges.length;
+    function setupCustomRangesWithCheckboxValues() {
+      new TestFixture()
+        .with(addNumericFacetWithRange({field, label}, numericRanges))
+        .init();
+    }
+
+    describe('verify rendering', () => {
+      before(setupCustomRangesWithCheckboxValues);
+      CommonFacetAssertions.assertDisplayFacet(NumericFacetSelectors, true);
+      CommonAssertions.assertAccessibility(numericFacetComponent);
+      CommonFacetAssertions.assertNumberOfSelectedCheckboxValues(
+        NumericFacetSelectors,
+        0
+      );
+      CommonFacetAssertions.assertNumberOfIdleCheckboxValues(
+        NumericFacetSelectors,
+        numberOfCustomRanges
+      );
+      CommonFacetAssertions.assertDisplayClearButton(
+        NumericFacetSelectors,
+        false
+      );
+    });
+
+    describe('when selecting a value', () => {
+      const selectionIndex = 1;
+      function setupSelectCheckboxValue() {
+        setupCustomRangesWithCheckboxValues();
+        selectIdleCheckboxValueAt(NumericFacetSelectors, selectionIndex);
+        cy.wait(TestFixture.interceptAliases.Search);
+      }
+
+      describe('verify rendering', () => {
+        before(setupSelectCheckboxValue);
+        CommonAssertions.assertAccessibility(numericFacetComponent);
+        CommonFacetAssertions.assertDisplayClearButton(
+          NumericFacetSelectors,
+          true
+        );
+        CommonFacetAssertions.assertNumberOfSelectedCheckboxValues(
+          NumericFacetSelectors,
+          1
+        );
+        CommonFacetAssertions.assertNumberOfIdleCheckboxValues(
+          NumericFacetSelectors,
+          numberOfCustomRanges - 1
+        );
+        CommonFacetAssertions.assertDisplayClearButton(
+          NumericFacetSelectors,
+          true
+        );
+      });
+
+      describe('verify analytics', () => {
+        before(setupSelectCheckboxValue);
+        NumericFacetAssertions.assertLogNumericFacetSelect(
+          field,
+          selectionIndex
+        );
+      });
+    });
+  });
+
+  describe('with custom #withInput only', () => {
+    const min = 0;
+    const max = 100000;
+    function setupRangesWithInputOnly() {
+      new TestFixture()
+        .with(
+          addNumericFacet({
+            field,
+            label,
+            'with-input': 'true',
```

Comment:
with-input should either be `integer` or `decimal`
![Screen Shot 2021-07-23 at 11 07 15 AM](https://user-images.githubusercontent.com/4923043/126802109-6fe2a9f2-b32d-4b7f-934d-f1852916a065.png)